app/models/concerns/account_interactions.rb

```diff
@@ -234,7 +234,7 @@ def lists_for_local_distribution
          .where('users.current_sign_in_at > ?', User::ACTIVE_DURATION.ago)
   end
 
-  private
+  private # rubocop:disable Lint/UselessAccessModifier
```

Comment:
Actually, it’s true. Methods can’t be made private inside modules so this keyword does nothing.

Reply:
Actually, it seems that there is a false detection because there is `private` in `class_methods`.
So instead of disabling it, I changed to adding `class_methods` to the `ContextCreatingMethods` option.

https://github.com/rubocop-hq/rubocop/issues/3422